### ISSUE TITLE
Enable --console_enabled=false option to be able to use -fh and -fj option

### ISSUE THREAD

**Issue by erickangMSFT** [vassilvk/slacker#22]
Currently @config.console_enabled property is used by Slacker but it seems it's always set to true. 
When running slacker with -fh and -fj, html and json output also contains text message. It will be great to get html and json result without additional text message so that the result can be used in other application.

```
WideWorldImporters (localhost)
.....

Finished in 0.05271 seconds (files took 0.1624 seconds to load)
5 examples, 0 failures

{"version":"3.6.0","examples":[{"id":"./spec/sample_1.rb[1:1]","description":"contains system tables","full_description":"My database contains system tables","status":"passed","file_path":"./spec/sample_1.rb","line_number":4,"run_time":0.008863,"pending_message":null},{"id":"./spec/sample_1.rb[1:2]","description":"contains system tables (take two)","full_description":"My database contains system tables (take two)","status":"passed","file_path":"./spec/sample_1.rb","line_number":10,"run_time":0.005719,"pending_message":null},{"id":"./spec/sample_1.rb[1:3]","description":"contains system tables (take three)","full_description":"My database contains system tables (take three)","status":"passed","file_path":"./spec/sample_1.rb","line_number":17,"run_time":0.005833,"pending_message":null},{"id":"./spec/sample_1.rb[1:4]","description":"can play with numbers","full_description":"My database can play with numbers","status":"passed","file_path":"./spec/sample_1.rb","line_number":23,"run_time":0.005111,"pending_message":null},{"id":"./spec/sample_1.rb[1:5]","description":"can play with numbers (take two)","full_description":"My database can play with numbers (take two)","status":"passed","file_path":"./spec/sample_1.rb","line_number":53,"run_time":0.024627,"pending_message":null}],"summary":{"duration":0.052712,"example_count":5,"failure_count":0,"pending_count":0},"summary_line":"5 examples, 0 failures"}
```

**Comment by erickangMSFT** [comment]
fix: https://github.com/erickangMSFT/slacker/commit/8e6b55ee5697d5ac860d7d0ae0a4fa9fe870deaa

**Comment by erickangMSFT** [comment]
![image](https://user-images.githubusercontent.com/11077282/29484013-961e7364-846a-11e7-84f4-7286e6145bc8.png)

**Comment by erickangMSFT** [comment]
![image](https://user-images.githubusercontent.com/11077282/29484014-a7cd2920-846a-11e7-9f70-2c2465259a88.png)

**Comment by erickangMSFT** [comment]
![image](https://user-images.githubusercontent.com/11077282/29484027-b9978632-846a-11e7-99f3-8ecba95bac2c.png)

**Comment by erickangMSFT** [comment]
![image](https://user-images.githubusercontent.com/11077282/29484029-c8e417f4-846a-11e7-87bc-580f10963826.png)

**Comment by erickangMSFT** [comment]
the current behavior is default without specifying any argument not to break existing slacker environment.

![image](https://user-images.githubusercontent.com/11077282/29484293-484509b8-8470-11e7-8414-73b81b599d28.png)

**Comment by vassilvk** [comment]
@erickangMSFT - this looks good.
Do you mind posting a pull request to master slacker?

P.S.
Regarding your comment about passing `console_enabled` as an argument, this will be resolved by https://github.com/vassilvk/slacker/issues/4.

**Comment by vassilvk** [comment]
Closed by #23.
